test/scripts/install-kind.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -ex
+
+export GO111MODULE="on"
+mkdir -p $GOPATH/bin
+curl -fs https://chunk.io/trozet/ba750701d0af4e2b94b249ab9de27b50 -o $GOPATH/bin/kubetest
```

Comment:
Something to explore for the future, is there a kind Docker image we could use (and mount-the-world into it) rather than relying on a prebuilt binary that Tim made at some point in the past?

Reply:
yeah we do caching now with github actions right @as-com ? Can we cache this built/binary as well?

Reply:
Yes, it definitely is feasible to build+cache kubetest like kubectl is right now.

Reply:
Opened #1215 as a reminder to address this.